pkg/process/procutil/process_linux_test.go

```diff
@@ -514,7 +475,6 @@ func testParseStatus(t *testing.T) {
 
 		assert.Equal(t, expMemInfo.RSS, actual.memInfo.RSS)
 		assert.Equal(t, expMemInfo.VMS, actual.memInfo.VMS)
-		assert.Equal(t, expMemInfo.Swap, actual.memInfo.Swap)
```

Comment:
This is deleted because I realized that gopsutil doesn't collect this field on function `MemoryInfo()`, so we don't compare it.